circuitpython_build_tools/scripts/build_bundles.py

```diff
@@ -269,8 +274,8 @@ def build_bundles(filename_prefix, output_directory, library_location, library_d
         for version in target_versions.VERSIONS:
             # Use prebuilt mpy-cross on Travis, otherwise build our own.
             if "TRAVIS" in os.environ:
-                mpy_cross = pkg_resources.resource_filename(
-                    target_versions.__name__, "data/mpy-cross-" + version["name"])
+                mpy_cross = importlib_resources.files(
+                    target_versions.__name__) / "data/mpy-cross-" + version["name"]
             else:
```

Comment:
@jepler Just to be explicitly clear, should this get removed?

Technically, I think it is dead code because the code later uses a pre-built mpy_cross hosted from S3 servers (IIRC).

Reply:
I'm fine with getting rid of TRAVIS references in the source code in this PR.